lighthouse-core/test/fixtures/invalid-audits/missing-generate-audit-result.js

```diff
@@ -9,9 +9,9 @@
 class MissingRequiredArtifacts {
   static get meta() {
     return {
-      name: 'missing-required-artifacts',
-      description: 'Missing required artifacts',
-      helpText: 'This is missing required artifacts',
+      id: 'missing-required-artifacts',
```

Comment:
maybe we should update these (and name of class) while we're noticing it?

Reply:
done